zerver/context_processors.py

```diff
@@ -160,6 +162,7 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         "root_domain_uri": settings.ROOT_DOMAIN_URI,
         "apps_page_url": get_apps_page_url(),
         "apps_page_web": apps_page_web,
+        "realm_is_self_hosted": realm_is_self_hosted,
```

Comment:
I'm going to blame my cold last week, but I realized we can use the `"corporate_enabled": corporate_enabled` context for these macros and don't need to add this context. Apologies for not catching this earlier.

So, `if corporate_enabled` is the same as `if not realm_is_self_hosted`.

Reply:
oh okay i'll just change it